### ISSUE TITLE
global name 'v' is not defined when headers missing

### ISSUE THREAD

**Issue by mwpastore** [TechEmpower/FrameworkBenchmarks#1784]
There is an issue with `toolset/benchmark/test_types/verifications.py` that manifests when mandatory response headers are missing from test results:

```
Accessing URL http://127.0.0.1:8080/db:
  Response (trimmed to 40 bytes): "{"id":5315,"randomNumber":3826}"
WARNING:root:Verifying test db for sinatra caused an exception: global name 'v' is not defined
   FAIL for http://127.0.0.1:8080

     Caused Exception in TFB
               This almost certainly means your return value is incorrect, 
               but also that you have found a bug. Please submit an issue
               including this message: global name 'v' is not defined
     Traceback (most recent call last):
       File "/FrameworkBenchmarks/toolset/benchmark/framework_test.py", line 389, in verify_type
         results = test.verify(base_url)
       File "/FrameworkBenchmarks/toolset/benchmark/test_types/db_type.py", line 50, in verify
         problems += verify_headers(headers, url, should_be='json')
       File "/FrameworkBenchmarks/toolset/benchmark/test_types/verifications.py", line 53, in verify_headers
         ('warn', 'Required response header missing: %s' % v, url))
     NameError: global name 'v' is not defined
     See http://frameworkbenchmarks.readthedocs.org/en/latest/Project-Information/Framework-Tests/#specific-test-requirements
```

This error seems occur when response headers are missing regardless of the framework being tested. Adding the headers resolves the issue; modifying this block of code in `verifications.py` to not try to reference `v` in the `sprintf` also resolves the issue:

```ruby
    if any(v.lower() not in headers for v in ('Server', 'Date', 'Content-Type')):
        problems.append(
            ('warn', 'Required response header missing: %s' % v, url))
```

I'm using the Vagrant development environment on the current tip.

**Comment by msmith-techempower** [comment]
Related to #1786

**Comment by msmith-techempower** [comment]
I believe this was resolved in #1786
